charts/argo-cd/README.md

```diff
@@ -64,7 +64,170 @@ applicationSet:
   replicas: 2
 ```
 
-### Synchronizing Changes from Original Repository
+## Ingress configuration
+
+Please refer to [Operator Manual](https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#ingress-configurationh) for details as samples below
+corespond to respetive sections.
+
+### SSL-Passthrough
+
+The `tls: true` option will expect that `argocd-server-tls` secret exists as Argo CD server loads TLS certificates from this place.
+
+```yaml
+certificate:
+  enabled: true
+  domain: argocd.example.com
+
+server:
+  ingress:
+    enabled: true
+    hostname: argocd.example.com
+    ingressClassName: nginx
+    annotations:
+      nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    tls: true
+```
+
+### SSL Termination at Ingress Controller
+
+```yaml
+configs:
+  params:
+    server.insecure: true
+
+server:
+  ingress:
+    enabled: true
+    hostname: argocd.example.com
+    ingressClassName: nginx
+    annotations:
+      nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    extraTls:
+      - hosts:
+        - argocd.example.com
+        # Secret might be optional based on used ingress controller
+        secretName: wildcard-tls
+```
+
+> **Note:**
+> It's also possible to use `tls: true` without `extraTls` if you don't plan to use wildcard certificate.
+
+### Multiple ingress resources for gRPC protocol support
+
+Use `ingressGrpc` section if your ingress controller supports only single protocol per Ingress resource (i.e.: Contour).
```

Comment:
1. should be "supports only a single"

2. should 'Ingress' have an uppercase I here? it's not consistent.

Reply:
1) Fixed.
2) I've used Ingress with capital I for K8s resource and ingress controller for deployment